### ISSUE TITLE
EPEERINVALID on installation

### ISSUE THREAD

**Issue by Toadskin** [angular/material2#849]
#### Bug, feature request, or proposal:

If those are my options it's a bug. 

This fails with EPEERINVALID using Node v4.4.7 and that has npm 2.15.8: 

install --save @angular2-material/core @angular2-material/button @angular2-material/card

#### What is the expected behavior?

Clearly, that installation should succeed. 

#### What is the current behavior?

It don't.

#### What are the steps to reproduce?

See bug description.

**Comment by snolflake** [comment]
Yesterday they updated everything to RC4 and are using latest version of the angular-cli, so maybe if you recreate package.json these errors go away. Otherwise just make sure that your angular installation is updated and simply ignore the version mismatch in the material packages (EPEERINVALID).

During alpha some components will have some outdated dependencies but they will be updated over time as someone is able to test is the changes doesn't break anything. 

**TL; DR**
Its just a matter of time to get the installation without any errors/warnings, but if your angular installation is OK you can just ignore them for now...

**Comment by jelbourn** [comment]
You likely can ignore peer dependency errors (they really should be warnings), or install the `@next` version.
